docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[configure-endpoint-integration-policy]]
+= Configure an integration policy
+
+After the {agent} is installed successfully with {endpoint-sec}, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum+ license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, and trusted applications to meet your organization's security needs.
+
+To configure the integration policy:
+
+1. In the {security-app}, go to **Manage** -> **Endpoints** to view the Endpoints list. Remember that you must have admin permissions in {kib} to access this page.
```

Comment:
I like the "admonition before" option, because it informs the reader before they get into the procedure.

But I think that it's still helpful to include the context in step 1 ("In the {security-app}"), mainly because the topics in the "Getting started" section aren't firmly rooted in _just_ the Security app. Some procedures are in Fleet, others are in Kibana advanced settings, etc.

Reply:
@joepeeples ah, now I see why you mentioned the Security app in step 1. Setting up the Endpoint integration, agents, etc. does require the user to exit the Security app so I agree that re-orienting them towards it is necessary for these steps. No objections to keeping the Security app bit in step one then.